lib/components/app/responsive-webapp.js

```diff
@@ -291,7 +291,7 @@ class RouterWrapperWithAuth0 extends Component {
             onLoginError={showLoginError}
             onRedirectCallback={processSignIn}
             onRedirecting={BeforeSignInScreen}
-            redirectUri={URL_ROOT}
+            redirectUri={window.location.origin}
```

Comment:
I think it's probably ok to remove URL_ROOT throughout the app, but this reminds me that we should document somewhere that user accounts/middleware is not currently compatible with hosting the OTP app at a path, e.g., https://newplanner.trimet.org/map. If we're not OK with that limitation we should probably fix it. (I think the only major concern is probably with the TriMet implementation.)

Reply:
That is a good point, so I should probably keep URL_ROOT and add a TODO for supporting a root path that has a subfolder.